commitlint.config.ts

```diff
@@ -168,6 +168,17 @@ module.exports = {
                     ];
                 },
 
+                'reject-square-bracket-style': ({header}: {header:any}) => {
+                    let headerStr = convertAnyToString(header, "header");
+
+                    let offence = headerStr.match(`^\\[.*\\]`) !== null
+
+                    return [
+                        !offence,
+                        `Please use "area/scope: title" style instead of wrapping area/scope under square brakets in your commit message header`
```

Comment:
@realmarv it should suggest area/scope or area(scope)

Reply:
Done